### PR TITLE
Dip-to-white animations between chromosomes

### PR DESCRIPTION
**Pull request by ens-ds23**
## Type

- [ ] Bug fix
- [X] New feature
- [ ] Improvement to existing feature

## Related JIRA Issue(s)
ENSWBSITES-402

## Importance
Typical

## Description
Adds a "dip to white" during chromosome switches to make it clearer what's happening for data that stays the same.

The focus objects sometimes jump confusingly in the middle of this, but that will be a different PR.

## Views affected
Genome Browser

### Other effects

- [ ] I have checked any requirements for Google Analytics
- [ ] I have created any required tests
- [X] The PR has as its final commit a WASM/JS update commit if any Rust files were updated.

## Possible complications

This PR is in the middle of a big block of PRs that were all merged at once. There's a moderate chance of some of them "stomping" over the others in the final asset, though I'll do my best to avoid that happening. If it does then a simple recompile on dev should fix it.

### CODE REVIEW

File: src/assets/browser/app/src/controller/scheduler/schedmain.rs [hunk 1]
```diff
@@ -66,7 +66,7 @@ impl SchedulerMain {
     fn check_tempo(&mut self, burst: bool) {
         let now = browser_time();
         self.jank.detect(burst,now/1000.);
-        self.set_timesig(self.jank.gear());
+        //self.set_timesig(self.jank.gear());
```
Comment (azangru) [inline src/assets/browser/app/src/controller/scheduler/schedmain.rs]: should this go away?

**Comment by azangru** [comment]
Name of PR seems to be auto-generated from the name of the branch. Please make sure that when it's squash-and-merged it gets a better name.

**Comment by ens-ds23** [comment]
Bother, missed your second comment. I need a better way of spotting these comments.

**Comment by ens-ds23** [comment]
Updated PR title.

File: src/assets/browser/app/src/controller/scheduler/schedmain.rs (hunk 1 above)
Comment (ens-ds23) [reply src/assets/browser/app/src/controller/scheduler/schedmain.rs]: This commented-out-code will go when the scheduler is reimplemented. At the moment this isn't clear, so I'll put a proper comment to that effect.

Comment (azangru) [reply src/assets/browser/app/src/controller/scheduler/schedmain.rs]: D'oh! I should have read the comment to the commented line! :man_facepalming:

**Pull request merged**